Makefile

```diff
@@ -1,14 +1,17 @@
 
-SHELL  ?= $(shell command -v bash 2>/dev/null)
-DOCKER ?= $(shell command -v docker 2>/dev/null)
-PANDOC ?= $(shell command -v pandoc 2>/dev/null)
+EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
+OUTPUT_DIRNAME		?= output/
```

Comment:
On Wed, Jun 01, 2016 at 11:58:45AM -0700, Vincent Batts wrote:
> > @@ -1,14 +1,17 @@
> >  
> > -SHELL  ?= $(shell command -v bash 2>/dev/null)
> > -DOCKER ?= $(shell command -v docker 2>/dev/null)
> > -PANDOC ?= $(shell command -v pandoc 2>/dev/null)
> > +EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
> > +OUTPUT_DIRNAME		?= output/
> 
> Well, no. They're similar, but not the same.

Right.  I think they should be unified into a single variable, but
that unification will take a bit of thinking to figure out.